src/esp32_platform.cpp

```diff
@@ -90,6 +90,7 @@ uint8_t * Esp32Platform::getEepromBuffer(uint16_t size)
 
 void Esp32Platform::commitToEeprom()
 {
+    EEPROM.getDataPtr();//make eeprom dirty again
```

Comment:
out of the library, make it dirty, always commit, then it's clean again. And out of the library, no one knows how the library work and need to make it dirty after commit. 
In other words, the library do need to write data to flash when calling commit(), just because of the DIRTY flag is cleared by operation out of library(which conflicts), so we just reset the flag to make it dirty again before commit(). And in my case, it seems no effect about writing 2048 byte data to flash.

Reply:
You shouldn't use EEPROM out of the lib at all. If you want to save something extra to flash, you can register an own SaveRestore at the knx Object. The bme680-demo should use this.